src/vm/i386/cgencpu.h

```diff
@@ -87,6 +87,10 @@ BOOL Runtime_Test_For_SSE2();
 // Needed for PInvoke inlining in ngened images
 #define HAS_NDIRECT_IMPORT_PRECODE              1
 
+#ifdef FEATURE_PAL
+#define USE_INDIRECT_CODEHEADER
```

Comment:
This should be left undefined (see my comment about changing the ABI in the other PR).